### ISSUE TITLE
TypeError: copytree() got an unexpected keyword argument 'dirs_exist_ok'

### ISSUE THREAD

**Issue by rainchen** [drizzleco/create-flask-app#5]
here is the exception detail :

```
Copying files...
Traceback (most recent call last):
  File "/Users/rain/.pyenv/versions/3.7.7/bin/create-flask-app", line 8, in <module>
    sys.exit(prompt_user())
  File "/Users/rain/.pyenv/versions/3.7.7/lib/python3.7/site-packages/create_flask_app/create_flask_app.py", line 112, in prompt_user
    copytree(resource_path(scaffold_path + folder), name + '/' + folder, dirs_exist_ok=True, copy_function=render_and_copy)
TypeError: copytree() got an unexpected keyword argument 'dirs_exist_ok'
```

this exception was from the code:

```
copytree(resource_path(scaffold_path + folder), name + '/' + folder, dirs_exist_ok=True, copy_function=render_and_copy)
```

but `copytree`  accepts a new `dirs_exist_ok` keyword argument was a [new change](https://docs.python.org/3/whatsnew/3.8.html#shutil) in python 3.8, should change the project requirement to require python 3.8+

should update setup.py as:

```
    python_requires='>=3.8',
```

**Comment by fangherk** [comment]
Oh, interesting, thanks for the find. @petabite can you add a quick fix?

**Comment by petabite** [comment]
Fixed! #6. Pushed new version to PyPI as well. Thanks for the catch @rainchen!
